external/libunwindstack/overrides/unwindstack/MapInfo.h

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
add comment for local changes in overriden files something like:

// ORBIT CHANGE START
...
// ORBIT CHANGE END

Reply:
Good point, I did it in a way that keeps the line count and position the same.